packages/valory/skills/abstract_round_abci/behaviours.py

```diff
@@ -156,6 +153,7 @@ class AbstractRoundBehaviour(
     abci_app_cls: Type[AbciApp[EventType]]
     behaviours: AbstractSet[BehaviourType]
     initial_behaviour_cls: BehaviourType
+    termination_behaviour_cls: Optional[BehaviourType]
```

Comment:
IMO it would make sense, for now, to keep this as `termination`. 

If we extend it in the future and support multiple background behaviours then we could rename the attribute.

Reply:
I argue for background below inside `abstract_round_abci` and for `termination` for the skill.